packages/gatsby-plugin-netlify/src/gatsby-node.js

```diff
@@ -32,5 +33,8 @@ exports.onPostBuild = async ({ store, pathPrefix }, userPluginOptions) => {
   const pluginData = makePluginData(store, assetsManifest, pathPrefix)
   const pluginOptions = { ...DEFAULT_OPTIONS, ...userPluginOptions }
 
-  return buildHeadersProgram(pluginData, pluginOptions)
+  const { redirects } = store.getState()
+
+  await buildHeadersProgram(pluginData, pluginOptions)
+  await createRedirects(pluginData, redirects)
```

Comment:
:neckbeard: Maybe this could be a promise.all, so they can run in parallel instead of series?